charts/generic-prometheus-alerts/ci/tests/ingress-alerts-tests.yaml

```diff
@@ -56,10 +56,10 @@ tests:
     name: 5xxErrorResponses alert not fired as path is for /health
 
     promql_expr_test:
-      - expr: avg(rate(nginx_ingress_controller_requests{exported_namespace="test-application-dev", ingress="test-application-ingress", path!="/health", status=~"5.*"}[5m]) > 0) by (ingress, exported_namespace)
+      - expr: avg(rate(nginx_ingress_controller_requests{exported_namespace="test-application-dev", ingress="test-application-ingress", path="/", status=~"5.*"}[5m]) > 0) by (ingress, exported_namespace)
         eval_time: 6m
         exp_samples:
-      - expr: avg(rate(nginx_ingress_controller_requests{exported_namespace="test-application-dev", ingress="test-application-ingress", path="/health", status=~"5.*"}[5m]) > 0) by (ingress, exported_namespace)
+      - expr: avg(rate(nginx_ingress_controller_requests{exported_namespace="test-application-dev", ingress="test-application-ingress", path="/health", status=~"5.*"}[5m]) > 0.004) by (ingress, exported_namespace)
```

Comment:
tested against current false positive alerts for `hmpps-prisoner-to-nomis-update-prod`.  They all seem to be around the `0.0037` mark so `0.004` will then ignore them

Reply:
also only doing the `/health` endpoint for now `/` is still at `0`